production/user_guide.md

```diff
@@ -99,27 +99,27 @@ A: The token will expire after twenty minutes.  The next time you use the BCDA A
 ### 1. Getting comfortable in Swagger
 There are two categories of information that you can retrieve through BCDA: metadata and bulk beneficiary data.
 
-<img class="ug-img" src="/assets/img/swagger_walkthrough_01.png" alt="swagger intro" />
+<img class="ug-img" src="/assets/img/swagger_walkthrough_01.png" alt="Swagger: 'metadata' and 'bulkData' are categories that can be expanded further to get more detailed information" />
 
 **Metadata** in BCDA includes information about the platform that is making, storing, and verifying credentials and tokens (the auth provider); information about the API’s version; and information about the actions you can perform using the API itself (also duplicatively termed metadata). There is no PII or PHI in the **metadata** endpoint, so you can access this endpoint without having to be authorized.
 
-<img class="ug-img" src="/assets/img/swagger_walkthrough_02.png" alt="swagger metadata intro" />
+<img class="ug-img" src="/assets/img/swagger_walkthrough_02.png" alt="" />
 
 ### 2. Looking at BCDA Metadata
 
 We’ll use `auth` as an example here.
 
 Under the Metadata endpoint, click on `/_auth` to expand that section. After the information field expands, as shown below, click `Try it out`.
 
-<img class="ug-img" src="/assets/img/swagger_walkthrough_03.png" alt="swagger metadata usage" />
+<img class="ug-img" src="/assets/img/swagger_walkthrough_03.png" alt="" />
 
 Then, as shown below, click `Execute` to run the process of getting details about `auth`.
 
-<img class="ug-img" src="/assets/img/swagger_walkthrough_04.png" alt="Screenshot of swagger metadata usage" />
+<img class="ug-img" src="/assets/img/swagger_walkthrough_04.png" alt="" />
 
 As shown below, clicking `Execute` returns details about the authorization and authentication provider BCDA is using.
 
-<img class="ug-img" src="/assets/img/swagger_walkthrough_05.png" alt="swagger metadata usage" />
+<img class="ug-img" src="/assets/img/swagger_walkthrough_05.png" alt="Swagger: the response body reveals that the auth_provider is 'alpha'" />
```

Comment:
Out of scope for this task... but is it time to replace this screenshot to show that `ssas` is returned?